### ISSUE TITLE
URL not showing up

### ISSUE THREAD

**Issue by lucasbrechot1** [google/physical-web#934]
Hi, 

I have deployed in partnership with a city organisation beacons equipped with physical web (Eddystone URLs) in the urban space in order to enable passengers to access citizen services. 
However although I can detect the beacons walking by, I do not detect the physical web urls on my phone. 

An example url is: https://iotct.io/ClRa8

Do you know what is causing this problem?

Thank you for letting me know

**Comment by ferencbrachmann** [comment]
Hey, your URL does not validate with this unofficial validator: https://beaufortfrancois.github.io/sandbox/physical-web/url-validator/

**Comment by ferencbrachmann** [comment]
Every resource on the target URL has to have a valid https certificate! I suggest trying a specialized tool like [Beeem](https://beeem.co) for your beacon landing pages.

**Comment by lucasbrechot1** [comment]
Hi! 

Thank you very much for your reply! I am not clear what you mean by "Every resource on the target URL has to have a valid https certificate", does it mean that every link included on the landing page needs to be https?

**Comment by bradmurray** [comment]
No, every URL that it forwards to until the final page has to be https. So if you use a https://bit.ly to forward to an http://xyz.com page it will not accept it. I assume that the final page has to be a fully valid https page meaning there can be no http images or javascript on it.

**Comment by ferencbrachmann** [comment]
@bradmurray You are not correct. Any URL can be HTTP until the final URL. And every resource on the final URL (for example: images, css) has to be HTTPs. You can put http links into the page, but the page content has to be exclusively https.

**Comment by lucasbrechot1** [comment]
@ferencbrachmann Understood! Thank you for the information!
